### DISCUSSION TITLE
In which version was the "noindex" attribute added to the "Account" API entity?

### DISCUSSION THREAD

**Discussion by pezcurrel** [mastodon/mastodon#22228] [Q&A] [answered]
Hello all,
i'd like to know in which version the "noindex" attribute was added to the "Account" API entity, since it is very useful, but not documented yet (https://docs.joinmastodon.org/entities/Account/).

**Comment by trwnh** [comment, marked as answer]
Added in v4.0.0 -- it's only available for local accounts. And it isn't all that useful currently -- its only use is for the web app to render a noindex tag on public pages for local accounts only. Third-party clients don't really have a use for this attribute, unless they serve public pages for some reason (and idk why they would, since mastodon doesn't support alternative frontends...)

As far as its exclusion from the docs, that is just an oversight since it was missing from the changelog as well.

**Reply by pezcurrel** [reply]
Thank you :)
We find it useful at mastodon.help because we are updating our mastodon instances crawler and now we can avoid fetching and displaying admin account info if they have noindex set to true (it's not currently implemented, but it will be soon). Also, we are experimenting around the possibility of making a "global mastodon public profiles directory", that would daily fetch accounts info from the directory api endpoint of every instance with ver. >= 4.0.0 and that honors noindex in addition to #nobot, #nobots, #noindex tags which may be present in the bio.
